### DISCUSSION TITLE
What 3d printers support PrettyGCode?

### DISCUSSION THREAD

**Discussion by Jeffv11** [Kragrathea/OctoPrint-PrettyGCode#125] [Q&A]
I have two Ender 3 V2 printers and have no issue issue with PGC. I also picked up a Think3dim T22 Max Pro+ printer and it appears that it is not supported by PGC. The Support Team at Think3dim provided me with Cura profiles and the profile has the G-code flavor set as Marlin.
What 3d printers support PrettyGCode?
TIA

**Comment by Kragrathea** [comment]
It should support all printers. Its just GCode as far as PGC is concerned. Does the built in Octoprint GCode preview work?

**Reply by Jeffv11** [reply]
Thx for the reply. No, the Octoprint GCode preview also does not display any graphics. Under GCode preview, I can select the Zoom entries from the dropdown, but no graphics are displayed. My next step is to review the log files for PrettyGCode and Preview GCode for errors.

**Reply by Kragrathea** [reply]
If the built in preview also isn't working it is probably an Octoprint issue. You may want to post something on the OP forums.

**Comment by Jeffv11** [comment]
K. Thx.
